src/core/emsesp.cpp

```diff
@@ -1810,10 +1811,14 @@ void EMSESP::shell_prompt() {
 
 // main loop calling all services
 void EMSESP::loop() {
-    uuid::loop();         // store system uptime
-    esp32React.loop();    // web services
-    system_.loop();       // does LED and checks system health, and syslog service
-    webLogService.loop(); // log in Web UI
```

Comment:
I did not find this weblog call in the new version.

Reply:
oops again! that is why I love pair-programming with you!